package.json

```diff
@@ -4,19 +4,19 @@
     "packages/*"
   ],
   "scripts": {
-    "build": "yarn run build:api && yarn workspace @oasis/web run build",
+    "build": "yarn workspace @oasis/web run build",
```

Comment:
Keep the build command as it is, "yarn build" is supposed to build both the API and Web, but people can specify what to build (ex. yarn build:api)